showroom.js

```diff
@@ -0,0 +1,69 @@
+//Problem : Create a application for a showroom showing 3 types of car having some common data names and some uncommon data names. Use inheritance to accomplish it.
+var Vehicle = function(data) {
+  this.name = data.name;
+  this.model = data.model;
+  this.make = data.make;
+  this.color = data.color;
+};
+
+var SUV = function(data) {
+  Vehicle.call(this, data);
+  this.airbag = data.airbag;
+  this.foglight = data.airbag;
+  this.type = "SUV";
+  this.avg = "11";
+};
+
+SUV.prototype = Vehicle.prototype;
+
+var Sedan = function(data) {
+  Vehicle.call(this, data);
```

Comment:
@abansal26 What would be the alternative of  'call'?

Reply:
The alternative for call will be apply here.